### PR TITLE
Integrate with Serverless Dashboard

### PR DESCRIPTION
**Pull request by eahefnawy**
This PR replaces our first version of the Serverless Platform by integrating into the new Serverless Dashboard.

### CODE REVIEW

**Comment by ac360** [comment]
So excited :)

File: lib/Serverless.js [hunk 1]
```diff
@@ -75,6 +76,24 @@ class Serverless {
...
+      && globalConfig.users[currentId]
+      && globalConfig.users[currentId].auth
+      && globalConfig.users[currentId].auth.id_token
+      && !globalConfig.users[currentId].dashboard) {
+      isTokenExpired = true;
+    }
+
+    if (isTokenExpired) {
+      this.cli
+        .log('WANRNING: Your login token has expired. Please run "serverless login" to login.');
```
Comment (pmuens) [inline lib/Serverless.js]: `WARNING`

File: docs/platform/README.md [hunk 2]
```diff
@@ -33,29 +33,50 @@ $ serverless -v
 
 ## Usage
```
Comment (ganeshvlrk) [inline docs/platform/README.md]: @eahefnawy Can you please remove everything except the login, logout and emit command and simply point users to the main serverless platform docs

File: .appveyor.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ install:
   # Get the version of Node.js
   - ps: Install-Product node $Env:NODEJS_VERSION
   # install modules
+  - rm -rf node_modules
   - npm install
```
Comment (vladgolubev) [inline .appveyor.yml]: Looks like `npm ci` can be used, it does both `rm -rf` + `npm install`

**Pull request merged**